tutorials/core-concepts/users-and-permissions/oauth-passport.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "OpenAuth: Google, Twitter, and Facebook Login"
```

Comment:
Globally replace OpenAuth with Single Sign-On, OpenAuth is a specific project/product we are not using

Reply:
Done.